packages/terra-core-docs/src/terra-dev-site/doc/content-container/About.1.doc.mdx

```diff
@@ -37,6 +38,12 @@ This component requires the following peer dependencies be installed in your app
 import ContentContainer from 'terra-content-container';
 ```
 
+<Notice variant="ux-recommendation" ariaLevel="3">
+
+As a best practice to ensure proper page structure, the header attribute should have text within heading tag (h3) and the footer attribute should have text within paragraph tag (p)
```

Comment:
> should the header's added for content container will always be `h3`

It is recommended to be h3 tag, I have updated the string according to the comment

Reply:
> I think there is miss understanding here !! there is no suggestion made from UX on usage specifications for header and footer prop. According to UX below are the recommendation provided by UX on Jira : UXPLATFORM-8357
> 
> ```
> The content container header heading should have a prop for consuming teams to set the appropriate heading level for where the heading falls within the page. 
> 
> The content container footer heading should not be a heading. It should be a paragraph of text. 
> 
> As is, the structure it creates being a heading sets the expectation that the header and the footer are siblings, but they are not. The footer is a child of the content container heading. 
> 
> Because the footer should not have other content below it, there is no need for it to be a lower-level heading than the main content container heading.
> ```

Updated